### DISCUSSION TITLE
Student Developer Pack Verification- Adding PDF Options

### DISCUSSION THREAD

**Discussion by kjeejercito** [community/community#129751] [GitHub Education]
### Hi everyone,

I am having issues verifying my account because I am trying to upload documents instead of my ID, but it requires a photo. I am also currently away from campus from the summer, but have added my school email and 2 factor identification and my billing address. Please consider allowing for a PDF to be uploaded with the consideration that most formal documents (ex. transcripts, letters of enrollment, etc.) are now online copies given a school email address. Taking a photo of an online document is an issue because the photo is either too large, or too low resolution.

**Comment by Astrobot-me** [comment]
Are you able to submit your application successfully?
